package.json

```diff
@@ -18,8 +18,8 @@
   },
   "homepage": "https://github.com/wapc/as-msgpack#readme",
   "devDependencies": {
-    "@as-pect/cli": "4.0.0",
-    "assemblyscript": "^0.17.1"
+    "@as-pect/cli": "6.0.x",
+    "assemblyscript": "0.18.x"
```

Comment:
With the old version contraints, it runs into error when `npm test`, due
to the fact that AS version constraint "^0.17.1" would allow fetching
the latest 0.18.x AS, which doesn't work well with the outdated as-pect.

```
  [TestFile.ts] node_modules/@as-pect/assembly/assembly/index.ts --binaryFile output.wasm --runtime stub --baseDir /home/vagrant/as-msgpack-di --debug --explicitStart --use ASC_RTRACE=1 --exportTable --importMemory --transform /home/vagrant/as-msg
pack-di/node_modules/@as-pect/core/lib/transform/index.js

ERROR TS2339: Property 'rtSize' does not exist on type '~lib/rt/common/BLOCK'.

     return changetype<BLOCK>(pointer - BLOCK_OVERHEAD).rtSize;
                                                        ~~~~~~
 in node_modules/@as-pect/assembly/assembly/internal/RTrace.ts(144,56)

[Error] There was a compilation error when trying to create the wasm binary for file: assembly/__tests__/msgpack.spec.ts.
[1 compile error(s)]
Error: exit 1
```

I'm updating both packages and locking the version to be available for patch releases.